src/ClientApp/src/components/BohrungForm.js

```diff
@@ -79,6 +94,24 @@ export default function BohrungForm(props) {
   const validateXCoordinate = (value) => value > 2590000 && value < 2646000;
   const validateYCoordinate = (value) => value > 1212000 && value < 1264000;
 
+  const validateDistance = () => {
+    let isValid = false;
+    if (currentStandort.bohrungen?.length === 0) {
+      isValid = true;
+    } else if (xCoordinate && yCoordinate) {
+      const src = "EPSG:2056";
+      const dest = "EPSG:4326";
+      const newCoordinates = transform([xCoordinate, yCoordinate], src, dest);
+      isValid = currentStandort.bohrungen
+        .map((b) => {
+          const existingCoordinates = transform([b.geometrie.coordinates[0], b.geometrie.coordinates[1]], src, dest);
+          return getDistance(newCoordinates, existingCoordinates);
+        })
+        .every((v) => v < 200);
```

Comment:
PP: `distance` anstatt `v` verwenden, zwecks Nachvollziehbarkeit.
```suggestion
        .every((distance) => distance < 200);
```